tests/unit/test_tablets.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+
+from cassandra.tablets import Tablets, Tablet
+
+
+class TabletsTest(unittest.TestCase):
```

Comment:
I'd like to see a few more test cases to verify various edge cases, right now this test class is quite small.
The best proof of that is that there currently is a bug in the algorithm, which should be easily caught by unit tests ;)